modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Popover.js

```diff
@@ -10,7 +10,7 @@
  */
 
 import ClayPopover from '@clayui/popover';
-import {Align} from 'metal-position';
+import {ALIGN_POSITIONS, align as alignElement} from 'frontend-js-web';
```

Comment:
Why call this here `as alignElement` when everywhere else it's simply `align`?

Reply:
Good catch, I  did that because the [`PopOver` component has an `align` prop](https://github.com/liferay-frontend/liferay-portal/blob/9478db18f8d8ff54c3d7b3206f6f995fc5048626/modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Popover.js#L33) that was conflicting with the `align` function

Reply:
👍